docs/sdks/client-sdks/react-native/initialization.mdx

```diff
@@ -76,6 +85,15 @@ The base URL for for the API. Can be overriden to proxy configuration requests.
 Poll for new configurations even if the initial configurations request failed.
 </ApiOptionRef>
 
+<ApiOptionRef
+  name="pollingIntervalMs"
```

Comment:
Proof that it is used in the init method

https://github.com/Eppo-exp/react-native-sdk/blob/96dcd2aebbf687d2804aec1bfef052c585f30f08/src/index.tsx#L107